pkg/client/keycloak/config.go

```diff
@@ -1,16 +1,22 @@
 package keycloak
 
 import (
+	"fmt"
 	"os"
 
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/environments"
 	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared"
 
 	"github.com/golang/glog"
 	"github.com/spf13/pflag"
 )
 
+type SSOProvider string
+
 const (
-	SSO_SPEICAL_MGMT_ORG_ID_STAGE string = "13640203"
+	MAS_SSO                       SSOProvider = "mas_sso"
```

Comment:
you can make this a string too.

That may avoid the need to `string(keycloak.MAS_SSO) `
```suggestion
	MAS_SSO                       string = "mas_sso"
```